package.json

```diff
@@ -82,16 +82,18 @@
   ],
   "jspm": {
     "main": "js/bootstrap",
+    "ignore": ["dist/js/npm"],
+    "directories": {
+      "lib": "dist"
+    },
     "shim": {
       "js/bootstrap": {
-        "deps": "jquery",
+        "deps": ["jquery"],
         "exports": "$"
       }
     },
-    "files": [
-      "css",
-      "fonts",
-      "js"
-    ]
+    "dependencies": {
+      "jquery": "github:components/jquery"
```

Comment:
This line tells jspm that bootstrap has a dep on jquery.  The fully qualified name for the jquery dep is `github:components/jquery`, which is consistent with what's in the [jspm registry](https://github.com/jspm/registry/blob/7171a47df3c0788b2add50710e68cd8f54f622c3/registry.json#L218).

CCing @guybedford in case I've got this wrong